router.go

```diff
@@ -167,6 +167,11 @@ func (app *App) handler(rctx *fasthttp.RequestCtx) {
 		return
 	}
 
+	// check flash messages
+	if c.Cookies("fiber_flash") != "" {
```

Comment:
not sure about the performance impact -> we should try to see this differently, as easily as possible
because with c.Cookies there will be a lot of allocations
and this is a general process for everyone